changes/ee/perf-12494.en.md

```diff
@@ -0,0 +1,5 @@
+Improve MongoDB connector performance.
+
+- improve mongodb-erlang client performance
+  [mongodb-erlang PR](https://github.com/emqx/mongodb-erlang/pull/41)
+- un-deprecate Mongodb `topology.pool_size` configuration parameter and set it to 10 by default.
```

Comment:
Maybe we could leave this out, since it's still hidden (undocumented) and only a lever for tuning if needed.

Reply:
Makes sense, removed.